release_scripts/Packages.Microsoft/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM amd64/ubuntu:18.04 AS builder
```

Comment:
You can mentioned there are issues with this. Does this work out of the box?

Reply:
The only issue I remember bringing up was the requirement for using amd64 to install the Repo Client. So with Docker, this should work out of the box.

Reply:
I thought you mention stdio was involved for a license agreement or something.

Reply:
Oh, yes, STDIO is required for responding to a prompt with the repo client installer. That's why the repo client installation is out of the dockerfile's scope—instead, the dockerfile completes requirements needed for repo client installation.

Reply:
I see. Can we have a similar script for those who don't use docker?

Reply:
If you don't use Docker, then it's just a matter of building a Linux machine that can install the repo client. If that's done, then the only remaining steps are:
1. Install Repo Client
2. Call the publish.sh file

No other scripts are needed.